python/test/unit/spacewalk/satellite_tools/test_yum_src.py

```diff
@@ -66,6 +68,10 @@ def _make_dummy_cs(self):
 
         return cs
 
+    @pytest.fixture(autouse=True)
+    def set_temp_path(self, tmpdir):
+        self.tmpdir = tmpdir.strpath
```

Comment:
I don't like this "additional `setUp` function", why is it needed? Isn't it possible to use the `tmpdir` fixture as lined out in the [pytest docs](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#use-fixtures-in-classes-and-modules-with-usefixtures)

Reply:
I just read more about this topic, it's true that `unittest.TestCase` type tests request fixtures ([docs](https://docs.pytest.org/en/7.1.x/how-to/unittest.html?highlight=unittest#pytest-features-in-unittest-testcase-subclasses)) or access them directly. Without migrating away from `unittest.TestCase` I currently see no way to improve this, but that's outside of the scope of this PR.